tests/lib/rules/id-length.js

```diff
@@ -63,7 +62,9 @@ ruleTester.run("id-length", rule, {
         { code: "var obj = { aaaaa: 1 };", options: [{ max: 4, properties: "never" }] },
         { code: "var obj = {}; obj.aaaaa = 1;", options: [{ max: 4, properties: "never" }] },
         { code: "({ a: obj.x.y.z } = {});", options: [{ max: 4, properties: "never" }], parserOptions: { ecmaVersion: 6 } },
-        { code: "({ prop: obj.xxxxx } = {});", options: [{ max: 4, properties: "never" }], parserOptions: { ecmaVersion: 6 } }
+        { code: "({ prop: obj.xxxxx } = {});", options: [{ max: 4, properties: "never" }], parserOptions: { ecmaVersion: 6 } },
+        { code: "var arr = [i,j,f,b]", parserOptions: { ecmaVersion: 6 } },
+        { code: "function foo([arr]) {}", parserOptions: { ecmaVersion: 6 }, errors: [tooShortError, tooShortError] }
```

Comment:
These errors probably shouldn't be here :)

Reply:
@mdjermanovic Thanks! Sorry for the mistake. I fixed it.